VMT/KCSA/KCSA-CVE-2020-2023.md

```diff
@@ -0,0 +1,57 @@
+announcement-date: 2020-06-12
+
+id: KCSA-CVE-2020-2023
+
+title: Kata Containers Execution with Unnecessary Privileges
+
+description: A container can access the guest root file system device.
+This can be used to gain code execution on the guest and masquerade as kata-agent.
+
+affected-components:
+
+  - components: `kata-agent`
+    version: Before v1.11.1
+
+vulnerabilities:
+
+  - CVE-ID: CVE-2020-2023
+
+reporters:
+
+  - name: `Yuval Avrahami`
+    affiliation: `Palo Alto Networks`
+    reported:
+      - CVE-2020-2023
+
+issues:
+
+  links:
+    - https://github.com/kata-containers/agent/issues/791
```

Comment:
Should probably also include https://github.com/kata-containers/runtime/issues/2476 and https://github.com/kata-containers/runtime/issues/2488. The matching pull requests are https://github.com/kata-containers/runtime/pull/2477 and https://github.com/kata-containers/runtime/pull/2487.